duckduckgo/iOS
# DuckPlayer - Add Pixel events in refactor

Task/Issue URL: https://app.asana.com/0/0/1208563377656004/f

**Description**:
- Updates refactored DuckPlayerHander with missing pixels
- Add tests for all relevant pixels



**Steps to test this PR**:
The update pixels include new unit tests, but if you want to test the implementation, follow these steps.

- Launch the app, Set DuckPlayer to 'Always'
- Hit Fire Button
- Search for Metallica videos
- [ ]  Confirm `duckplayer.view-from.serp` is fired

- Hit Fire Button
- Go to youtube.com
- Search for metallica
- Tap a video
- [ ]  Confirm `Pixel fired duckplayer.view-from.youtube.automatic` is fired

- Hit Fire Button
- Go to https://randomtests.netlify.app/ddg/duckplayer.html
- Tap "Regular Youtube link"
- [ ]  Confirm `Pixel fired duckplayer.view-from.other` is fired
